lib/whitehall.rb

```diff
@@ -32,70 +32,70 @@ def self.support_url
   end
 
   def self.available_locales
-    [
-      :en,
-      :ar,
-      :az,
-      :be,
-      :bg,
-      :bn,
-      :cs,
-      :cy,
-      :da,
-      :de,
-      :dr,
-      :el,
-      :es,
-      "es-419",
-      :et,
-      :fa,
-      :fi,
-      :fr,
-      :gd,
-      :gu,
-      :he,
-      :hi,
-      :hr,
-      :hu,
-      :hy,
-      :id,
-      :is,
-      :it,
-      :ja,
-      :ka,
-      :kk,
-      :ko,
-      :lt,
-      :lv,
-      :ms,
-      :mt,
-      :nl,
-      :no,
-      :pa,
-      :pl,
-      :ps,
-      :pt,
-      :ro,
-      :ru,
-      :si,
-      :sk,
-      :sl,
-      :so,
-      :sq,
-      :sr,
-      :sv,
-      :sw,
-      :ta,
-      :th,
-      :tk,
-      :tr,
-      :uk,
-      :ur,
-      :uz,
-      :vi,
-      :zh,
-      "zh-hk",
-      "zh-tw",
+    %i[
```

Comment:
Love this first commit. Alphabetising, making type consistent, and testing that the files are in sync. Well done 👏